src/bp/core/services/action/action-service.ts

```diff
@@ -173,7 +173,10 @@ export class ScopedActionService {
         session: incomingEvent.state.session,
         args: actionArgs,
         printObject: printObject,
-        process: _.pick(process, 'HOST', 'PORT', 'EXTERNAL_URL', 'PROXY')
+        process: {
+          ..._.pick(process, 'HOST', 'PORT', 'EXTERNAL_URL', 'PROXY'),
+          env: _.pickBy(process.env, (value, name) => name.startsWith('EXPOSED_'))
```

Comment:
What do you think of extracting those variables in a Memoized function to avoid looping through all env variables for each incoming event?

Reply:
I'll add it as a comment so we can merge now